data/hero-banner.yml

```diff
@@ -1,4 +1,4 @@
-isActive: true
+isActive: false
```

Comment:
Just wanted to check, am I correct that we want to stop showing banner?
If so, do we need to have the disabled banner in the code for future purposes or is it better to get rid of it completely for the sake of simplicity?